resources/scss/ceres/widgets/Header/_language-detection-widget.scss

```diff
@@ -0,0 +1,30 @@
+.language-detection{
+  &.widget-primary {
+      @include widget-bg-variant(rgba($primary, .8));
+  }
+
+  &.widget-secondary {
+      @include widget-bg-variant(rgba($secondary, .8));
+  }
+
+  &.widget-success {
+      @include widget-bg-variant(rgba($success, .8));
+  }
+
+  &.widget-info {
+      @include widget-bg-variant(rgba($info, .8));
+  }
+
+  &.widget-warning {
+      @include widget-bg-variant(rgba($warning, .8));
+  }
+
+  &.widget-danger {
+      @include widget-bg-variant(rgba($danger, .8));
+  }
```

Comment:
This isn't required. You can just add the class `bg-appearance` to apply the background color depending on the appearance-class

Reply:
@Tim-M-S can this be removed regarding to your changes on the widget template?